src/components/Controls.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styles from './Styles.css';
+
+function Controls({ header, headerColor, headerFont, footer, footerColor, footerFont, imgUrl, onChange }) {
+  return (
+    <form className={styles.form}>
```

Comment:
```suggestion
    <form onSubmit={onSubmit} className={styles.form}>
```
If you moved the button within the form, you would then use the onSubmit event.